.github/workflows/comment-command.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - name: React Rocket
-        uses: actions/github-script@v6
```

Comment:
Last time I did this, it broke the `/format` command. I just tried running this action, but it ran v6 instead, so you'll need to test this on your own branch.

Reply:
I've tested this on my teams repository with the react script it appears to work

Reply:
Could you link to the Actions run, and the comment it left a reaction on?

Reply:
https://github.com/spacey-sooty/2024-Crescendo/blob/master/.github/workflows/comment-command.yml here is the relevant file

https://github.com/spacey-sooty/2024-Crescendo/pull/15 here is the pr

https://github.com/spacey-sooty/2024-Crescendo/pull/15#issuecomment-1914704914 here is the comment

Reply:
Here's the Actions run: https://github.com/spacey-sooty/2024-Crescendo/actions/runs/7696893087/job/20972686941?pr=15

Reply:
Is anything about that not ok?

Reply:
Seems fine to me.